cmd/deploy.go

```diff
@@ -154,6 +147,14 @@ generates a deployment manifest (yaml) file if one is not present, and uses it t
 	},
 }
 
+func closeAndRemoveFile(file *os.File, fileName string) {
+	file.Close()
+	Debug.log("Removing file: ", fileName)
+	err := os.Remove(fileName)
+	if err != nil {
+		Warning.logf("Failed to remove %s.", fileName)
+	}
+}
 func deployWithKnative(cmd *cobra.Command, args []string) error {
```

Comment:
is this function for all deploys? or just for knative?

Reply:
the close and remove function is only for the non knative path

Reply:
So the reason we could do away with it was there was only one file to copy and it was not a symlink.  In the case of build/extract they are trying to copy an entire stack in which there can be symlinks.
So for just app-deploy.yaml we can live without the windows specific code.
But for a whole tree of a stack image, we'd need it.  So it needs to remain in the build and deploy path.
Rather than fight the battle of why the Windows path didn't work (my guess permissions and the /tmp directory), I elected to remove it, once I realized we could live without it for this cp of app-deploy.yaml.